src/applications/mhv-medical-records/util/txtHelpers/labsAndTests.js

```diff
@@ -14,8 +14,9 @@ ${
 
             details.push('Details about this test:', ' ');
 
-            if (record.sampleTested)
+            if (record.sampleTested && record.type !== 'pathology')
               details.push(`Site or sample tested: ${record.sampleTested}`);
+            else details.push(`Site or sample tested: ${record.sampleFrom}`);
```

Comment:
I would make this another separate condition instead of an `else`.

Also remember that you need to check for the presence of the thing you are displaying. So maybe something like:

```
if (record.sampleFrom && record.type === 'pathology')
  details.push(`Site or sample tested: ${record.sampleFrom}`);
```